### PR TITLE
[Serving] Add missing explain method

### PR DESCRIPTION
**Pull request by yanburman**
https://jira.iguazeng.com/browse/ML-3222 part 1 only

### CODE REVIEW

File: mlrun/frameworks/_ml_common/pkl_model_server.py [hunk 1]
```diff
@@ -56,3 +58,13 @@ def predict(self, request: dict) -> list:
...
+
+    def explain(self, request: Dict[str, Any]) -> str:
+        """
+        Return a string explaining what model is being serve in this serving function and the function name.
+
+        :param request: A given request.
+
+        :return: Explanation string.
+        """
+        return f"The '{self.model.name}' model serving function named '{self.name}'"
```
Comment (guy1992l) [inline mlrun/frameworks/_ml_common/pkl_model_server.py:70]: pickle model server is used for `lightgbm`, `scitki-learn` and `xgboost` which all don't have `name` property in their models. You can simply write something like:

```suggestion
        return f"A model server named '{self.name}'"
```

**Review by guy1992l — CHANGES_REQUESTED** [review]

File: mlrun/frameworks/_ml_common/pkl_model_server.py (hunk 1 above)
Comment (liranbg) [reply mlrun/frameworks/_ml_common/pkl_model_server.py:70]: @guy1992l - is there a point adding UT for coverage? or model could be arbitrary?

Comment (guy1992l) [reply mlrun/frameworks/_ml_common/pkl_model_server.py:70]: Not really... we do need to think of a way to test all of these though per framework like I did for training. It does require some work, but there is no need for special unit tests for this class.

File: mlrun/frameworks/_ml_common/pkl_model_server.py [hunk 2]
```diff
@@ -56,3 +58,13 @@ def predict(self, request: dict) -> list:
         y_pred: np.ndarray = self.model.predict(x)
 
         return y_pred.tolist()
+
+    def explain(self, request: Dict[str, Any]) -> str:
+        """
+        Return a string explaining what model is being serve in this serving function and the function name.
```
Comment (jillnogold) [inline mlrun/frameworks/_ml_common/pkl_model_server.py:64]: Returns a string listing the model that is being served in this serving function and the function name.

**Review by guy1992l — APPROVED** [review]

**Pull request merged**